diff -U3 -r --new-file pexts.orig/src/PCRE/configure.ac pexts/src/PCRE/configure.ac
--- pexts.orig/src/PCRE/configure.ac	2003-03-31 02:33:35.000000000 +0200
+++ pexts/src/PCRE/configure.ac	2003-05-25 03:43:32.000000000 +0200
@@ -4,7 +4,7 @@
 CAUDIUM_MODULE_INIT([PCRE])
 
 AH_BOTTOM([
-#include "pcrelib/pcre.h"
+#include <pcre.h>
 #define THIS ((PCRE_Regexp *)Pike_fp->current_storage)
 
 typedef struct
@@ -22,9 +22,14 @@
 static void free_regexp(struct object *);
 ])
 
-AC_CHECK_HEADERS(  locale.h)
-AC_CHECK_FUNCS( setlocale )
-AC_CONFIG_SUBDIRS(pcrelib)
+PCRE_LIBS="`pcre-config --libs`"
+PCRE_CFLAGS="`pcre-config --cflags`"
+
+AC_SUBST(PCRE_LIBS)
+AC_SUBST(PCRE_CFLAGS)
+
+AC_CHECK_HEADERS(locale.h pcre.h)
+AC_CHECK_FUNCS(setlocale )
 
 AC_SUBST(makefile_rules)
 AC_OUTPUT(Makefile)
diff -U3 -r --new-file pexts.orig/src/PCRE/Makefile.in pexts/src/PCRE/Makefile.in
--- pexts.orig/src/PCRE/Makefile.in	2003-03-31 02:14:43.000000000 +0200
+++ pexts/src/PCRE/Makefile.in	2003-05-25 03:42:27.000000000 +0200
@@ -1,17 +1,15 @@
 VPATH=@srcdir@:@srcdir@/../..:../..
 MODNAME=PCRE
-OBJS= pcre.o pcrelib/libpcre.a
+OBJS= pcre.o
 SOURCES=pcre.c
-MODULE_LIBS=@LIBS@
+MODULE_CFLAGS=@PCRE_CFLAGS@
+MODULE_LIBS=@LIBS@ @PCRE_LIBS@
 EXTRA_CLEAN=pcre_config.h pcre_config.h.in PCRE.so
 
 include @makefile_rules@
 
 extrainstall:
 
-pcrelib/libpcre.a: pcrelib/config.h pcrelib/dftables.c pcrelib/get.c pcrelib/internal.h pcrelib/maketables.c pcrelib/pcre.c pcrelib/pcre.h pcrelib/pcretest.c pcrelib/study.c
-	(cd pcrelib ; ${MAKE} )
-
 # DO NOT DELETE THIS LINE -- make  depend  depends  on  it.
 
 
