diff -U3 --new-file -r pexts.orig/src/PCRE/configure.ac pexts/src/PCRE/configure.ac
--- pexts.orig/src/PCRE/configure.ac	2002-12-26 21:19:21.000000000 +0100
+++ pexts/src/PCRE/configure.ac	2002-12-26 21:39:52.000000000 +0100
@@ -4,7 +4,7 @@
 CAUDIUM_MODULE_INIT()
 
 AH_BOTTOM([
-#include "pcrelib/pcre.h"
+#include <pcre.h>
 #define THIS ((PCRE_Regexp *)Pike_fp->current_storage)
 
 typedef struct
@@ -22,9 +22,14 @@
 static void free_regexp(struct object *);
 ])
 
-AC_CHECK_HEADERS(  locale.h)
-AC_CHECK_FUNCS( setlocale )
-AC_CONFIG_SUBDIRS(pcrelib)
+PCRE_LIBS="`pcre-config --libs`"
+PCRE_CFLAGS="`pcre-config --cflags`"
+
+AC_SUBST(PCRE_LIBS)
+AC_SUBST(PCRE_CFLAGS)
+
+AC_CHECK_HEADERS(locale.h pcre.h)
+AC_CHECK_FUNCS(setlocale )
 
 AC_OUTPUT(Makefile.pre, [${MAKE:-make} -f Makefile.pre depend])
 
